config/sequelize.js

```diff
@@ -51,6 +51,10 @@ db.User = User;
 db.UserMessage = UserMessage;
 db.UserThread = UserThread;
 
+//Run sql command to add new column, update lastMessageId column for those who are using Messaging API already
+sequelize.query('ALTER TABLE "UserThreads" DROP COLUMN IF EXISTS "isLog"; ALTER TABLE "Threads" ADD COLUMN IF NOT EXISTS "logUserId" integer DEFAULT NULL; ALTER TABLE "Threads" ADD COLUMN IF NOT EXISTS "lastMessageId" INTEGER; UPDATE "Threads" T1 SET "lastMessageId" = T2."MessageId" FROM ( SELECT max(id) "MessageId", "ThreadId" FROM "Messages" Group By "ThreadId" ) T2 WHERE T1."id" = T2."ThreadId" and "lastMessageId" is null;');
```

Comment:
I think this is great! So we don't have to drop and recreate databases just to add migrations. I would  be great to have this called from a separate migration file with subsequent migrations. I can see this file getting bloated with sequel code pretty quickly :).

Reply:
Agreed, it was a simple solution for time being. The migration file will be a more of a time consuming process, but necessary in the future.